src/webview/feedback/json.js

```diff
@@ -0,0 +1,89 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+export const json = {
+    'pages': [
+        {
+            'name': 'Feedback Page',
+            'elements': [
+                {
+                    'type': 'rating',
+                    'name': 'Satisfaction',
+                    'isRequried': true,
+                    'title': 'Overall, how satisfied or dissatisfied are you with the extension?',
+                    'rateDisplayMode': 'smileys',
```

Comment:
Let's use numbers 1-5 to rate. As the smiley with heart eyes can only be used for 2-10 rating. For 1-5 rating, it does not represent the correct smileys as expected.